src/iexamine.cpp

```diff
@@ -965,7 +966,7 @@ class atm_menu
  */
 void iexamine::atm( Character &you, const tripoint & )
 {
-    atm_menu {you}.start();
+    atm_menu {you} .start();
```

Comment:
```suggestion
    atm_menu {you}.start();
```

Reply:
This one's unrelated to my changes and I've gone to lengths to leave it alone based on the expectation that another PR will address it.

Reply:
It's showing up in your diff, so it's up to you to change it in your branch and prevent it from being someone else's problem.
https://github.com/CleverRaven/Cataclysm-DDA/blob/d7bcff2c2fcd22581273eb8b2f4e1caa61e60199/src/iexamine.cpp#L968

Reply:
Ah, I see.  This diff is actually a result of applying the CDDA standard `astyle` formatting rules to `iexamine.cpp`.  Notably, the `astyle` command line recommended by the docs is in conflict with the automated review tools.  Contention over this space has been causing tests to fail on all the PRs lately.

Reply:
Oh, really? I did need to install an older version of astyle to get styling working, so I guess there's something up there.